patch: clarifying offline vs inactive (#2334)
balena-io/docs: pages/learn/manage/billing.md

## Inactive vs Offline Devices

> To understand the difference between Inactive and Offline, we define them the following way:

Reviewer: ```suggestion
To understand the difference between inactive and offline devices, we define them in the following ways:
```

> - Inactive: The device has been deactivated, or has been preregistered but has not yet connected to the balenaCloud API.

Reviewer: ```suggestion
- **Inactive**: The device has been either [deactivated](https://www.balena.io/docs/learn/manage/actions/#deactivate-device) or [preregistered](https://www.balena.io/docs/learn/more/masterclasses/advanced-cli/#52-preregistering-a-device) but hasn't connected to the balenaCloud API yet.
```

> - Offline: The device is not connected to cloudlink and has not had any recent API communications.

Reviewer: ```suggestion
- **Offline**: The device is not connected to cloudlink and didn't have any recent API communications.
```

> If you have a device offline _and_ active you **will still be billed for that device.** Those devices are usually still deployed in the field and ready to be used at any time. If you make the decision to have your devices offline intentionally, you will still see them in your fleet and and can take some actions, such as applying updates, that will take effect as soon as they come online.

Reviewer: ```suggestion
If you have a device **offline and active**, you will **still be billed for that device**. Those devices are usually still deployed in the field and ready to be used at any time. If you make the decision to have your devices offline intentionally, you will still be able see them in your fleet and can take some actions such as applying updates, that will take effect as soon as they come online.
```

> If you have a device offline _and_ active you **will still be billed for that device.** Those devices are usually still deployed in the field and ready to be used at any time. If you make the decision to have your devices offline intentionally, you will still see them in your fleet and will have full visibility to remotely monitor the devices or to update them, actions that will take effect as soon as they become online.

Reviewer 2: > you will still see them in your fleet and will have full visibility to remotely monitor the devices or to update them, actions that will take effect as soon as they become online.

This seems a but confusing, maybe something like this instead?
```suggestion
If you have a device offline _and_ active you **will still be billed for that device.** Those devices are usually still deployed in the field and ready to be used at any time. If you make the decision to have your devices offline intentionally, you will still see them in your fleet and and can take some actions, such as applying updates, that will take effect as soon as they come online.
```

> Once you deactivate a device, you will not be able to monitor nor apply updates to it, and after the de-activation fee you won't be charged again until they come online.

Reviewer: ```suggestion
Once you deactivate a device, you will not be able to monitor nor apply updates to that device. After the de-activation fee is charged, you won't be charged again until the device comes back online.
```

docs always explicit

> Once you deactivate a device, you will not be able to monitor nor to apply updates to it, and you won't be charged after the de-activation fee until they become online again.

Reviewer 2: ```suggestion
Once you deactivate a device, you will not be able to monitor nor apply updates to it, and after the de-activation fee you won't be charged again until they come online.
```
